mods/ra2/rules/defaults.yaml

```diff
@@ -392,6 +392,7 @@
 		Voice: Attack
 	ActorLostNotification:
 	BodyOrientation:
+		QuantizedFacings: 32
```

Comment:
With voxels are you talking about all voxels or voxel aircraft, becuase i copied it from `^Vehicles` at here.